.github/workflows/tests.yaml

```diff
@@ -36,15 +62,22 @@ jobs:
           poetry-version: "1.3.2"
       - name: Install dependencies
         run: poetry install
-      - name: Test formatting
-        run: poetry run black --check .
-      - name: Test sorting
-        run: poetry run isort --check .
-      - name: Test docstring formatting
-        run: poetry run docformatter --check --config ./pyproject.toml .
       - name: Test with pytest
         run: poetry run pytest .
-      - name: Static code analysis
-        run: poetry run mypy .
-      - name: Run pylint
-        run: poetry run pylint quantuminspire
+  complete:
```

Comment:
Optionally we could replace this with https://github.com/upsidr/merge-gatekeeper which would eliminate the need to update this step when new jobs are added.